var/spack/repos/builtin/packages/edm4hep/package.py

```diff
@@ -32,7 +33,8 @@ class Edm4hep(CMakePackage):
     depends_on('python', type='build')
 
     depends_on('root@6.08:')
-    depends_on('podio@0.13:')
+    depends_on('podio@0.14:', when='@0.4:')
+    depends_on('podio@0.13.0:0.13', when='@:0.3')
```

Comment:
:see_no_evil:

Reply:
@alalazo This is the new default way of expressing `0.13.X` right? Unfortunately a lot of hep packages have the convention of omitting the patch release number if it is 0, so `0.13` is ambiguous in that it both refers  to `0.13.0` and `0.13.X`. I'd be happy for suggestions how to improve that ...